src/main/kotlin/org/jetbrains/research/ml/ast/storage/StorageKey.kt

```diff
@@ -0,0 +1,3 @@
+package org.jetbrains.research.ml.ast.storage
```

Comment:
It a little bit strange and inconvenient to have such a short class and to separate StorageKey from MetaDataStorage, why don't put it to the MetaDataStorage.kt?